### PR TITLE
Paintroid-727 : Make the gesture navigation bar color match the background color

### PR DESCRIPTION
**Pull request by vishad-tyagi**
Ticket: Paintroid-727
https://catrobat.atlassian.net/browse/PAINTROID-727

Description
The color of the gesture navigation bar does not match the app's background color, which leads to a lack of visual consistency and a less seamless user experience.

## New Features and Enhancements
- I have implemented SystemChrome.setSystemUIOverlayStyle to set the navigation bar color to transparent, and used SystemChrome.setEnabledSystemUIMode with SystemUiMode.edgeToEdge to enable drawing content behind the system navigation bar for a more immersive UI experience.

ScreenShot

## Refactorings and Bug Fixes
- Refactorings and Bug Fixes

#### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -23,6 +24,11 @@ void main() async {
   final prefs = await SharedPreferences.getInstance();
   final showOnboarding = prefs.getBool('showOnboarding') ?? true;
 
+  SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
```
Comment (msesko) [inline lib/main.dart:27]: This should be resolved later when my custom theme ticket gets merged. I am still waiting for someone to review it tho...

Comment (vishad-tyagi) [reply lib/main.dart:27]: ok that's good
So should I close the PR or leave it here for now?

Comment (msesko) [reply lib/main.dart:27]: change it to draft PR for now please :)

**Comment by juliajulie95** [comment]
Please resolve conflicts

**Pull request closed without merging**